contract/pkg/mock/ddc_bucket_contract_mock.go

```diff
@@ -4,15 +4,18 @@ import (
 	"errors"
 	"github.com/centrifuge/go-substrate-rpc-client/v2/types"
 	"github.com/cerebellum-network/cere-ddc-sdk-go/contract/pkg"
+	patractTypes "github.com/patractlabs/go-patract/types"
 	log "github.com/sirupsen/logrus"
 	"math/big"
 	"time"
 )
 
+var writerIds = getAccountIDs([]string{"5FJDBC3jJbWX48PyfpRCo7pKsFwSy4Mzj5t39PfXixD5jMgy"})
+
 var buckets = []*pkg.BucketStatus{
-	CreateBucket(1, `{"replication":1}`),
-	CreateBucket(2, `{"replication":2}`),
-	CreateBucket(3, `{"replication":3}`),
+	CreateBucket(1, `{"replication":1}`, writerIds),
+	CreateBucket(2, `{"replication":2}`, writerIds),
+	CreateBucket(3, `{"replication":3}`, writerIds),
```

Comment:
Will work other tests with mock (JS, Kotlin SDKs)?

Reply:
extended writer ids list to support accounts that are used in JS and Kotlin SDKs